presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -290,6 +301,27 @@ public boolean isLimitGuaranteed()
         return true;
     }
 
+    private static ColumnMapping timestampWithTimeZoneColumnMapping()
+    {
+        return ColumnMapping.longMapping(
+                TIMESTAMP_WITH_TIME_ZONE,
+                (resultSet, columnIndex) -> {
+                    // PostgreSQL does not store zone information in "timestamp with time zone" data type
+                    long millisUtc = resultSet.getTimestamp(columnIndex).getTime();
```

Comment:
Is this correct if the JVM time zone is not UTC?

Reply:
We run tests with non-UTC JVM zone (Bahia Banderas).

`java.sql.Timestamp` has dual-role
* it can be used to represent local time (best when used via `Timestamp#valueOf(java.time.LocalDateTime)` & `Timestamp#toLocalDateTime`)
* it can be used to represent point in time (best when used via `Timestamp#Timestamp(long)` and `Timestamp#getTime`)

Here we're leveraging this "second role".